decidim-core/app/cells/decidim/card_m/header.erb

```diff
@@ -14,4 +14,5 @@
   <% end %>
 
   <%= render :author if has_author? %>
+  <%= cell("decidim/coauthorships", model, has_actions: has_actions?) if coauthorable? %>
```

Comment:
Can a resource be both `Authorable` and `Coauthorable`?  If they can't, then I'd join lines 16 and 17 and make the cell keep the logic on how to render the authors.

Reply:
Something like:

```ruby
def author
  return render :author if has_author?
  return cell("decidim/coauthorships", model, has_actions: has_actions?) if coauthorable?
  nil
end
```

And then on the views replace lines 16-17 with `<%= author %>`

Reply:
@mrcasals I agree